README.md

```diff
@@ -1,25 +1,17 @@
 # RPM for drush
 
-This repository builds an RPM for drush.
+This repository builds RPMs for drush.
 
-## Relevant drush RPM names
-
-- drush: Legacy RPM containing drush and site-audit
-- drush-0.x: RPM containing only drush (version 0.x)
-- site-audit-0.x: RPM containing only site-audit 0.x
-
-The RPM filename built by this repository is:
+The RPM filenames built by this repository are formatted:
 ```
-drush-0.x-release-0.22.0-01458238016.git4602714.x86_64.rpm
-{  name  }-{ type}-{vers}-{iteration}.{ commit }.{arch}.rpm
+drush-x.x-release-8.1.2-1466798938.df43fcf.x86_64.rpm
```

Comment:
Every other component uses example numbers (e.g. 8.1.2 for vers); make the name component `drush-8.x` for consistency.